nvi-commons/src/main/java/no/sikt/nva/nvi/common/utils/ExceptionMapper.java

```diff
@@ -43,13 +44,14 @@ public static <T> ApiGatewayException map(Failure<T> failure) {
             logger.warn("Attempted operation which was not allowed", exception);
             return new MethodNotAllowedException(exception.getMessage());
         }
-        logger.error("BadGatewayException", exception);
-        return new BadGatewayException(exception.getMessage());
+        logger.error("BadGatewayException", exception.getMessage());
+        return new BadGatewayException("Something went wrong! Contact application administrator.");
```

Comment:
It is strange that we serve "default" exception for NotFoundException to client, but don't do it for BadGateway exception. Logging initial exception but returning "default" contact administrator message